scipy/stats/stats.py

```diff
@@ -409,8 +409,9 @@ def mode(a, axis=0, nan_policy='propagate'):
         Axis along which to operate. Default is 0. If None, compute over
         the whole array `a`.
     nan_policy : {'propagate', 'raise', 'omit'}, optional
-        Defines how to handle when input contains nan. 'propagate' returns nan,
-        'raise' throws an error, 'omit' performs the calculations ignoring nan
+        Defines how to handle when the input contains nan. 'propagate' 
+        attempts to performs calculations keeping nan values,'raise'
+        throws an error,'omit' performs the calculations ignoring nan
```

Comment:
there's missing spaces before 'raise' and 'omit'. 

textual change looks good to me

Reply:
OK, corrected :)